hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -369,14 +362,11 @@ private void displayValidationMessage(final StringBuilder errorMessage, final St
 
         private void addNewTarget(final String controllerId, final String name) {
             final String newControllerId = HawkbitCommonUtil.trimAndNullIfEmpty(controllerId);
-            if (mandatoryCheck(newControllerId) && duplicateCheck(newControllerId)) {
-                final String newName = HawkbitCommonUtil.trimAndNullIfEmpty(name);
-                final String newDesc = HawkbitCommonUtil.trimAndNullIfEmpty(descTextArea.getValue());
-
-                final Target newTarget = targetManagement.createTarget(entityFactory.target().create()
-                        .controllerId(newControllerId).name(newName).description(newDesc));
+            if (isNotDuplicate(newControllerId)) {
```

Comment:
Is this check necessary? The createTarget(…) method also hast the duplicate check. You cloud just catch the EntityAlreadyExistsException and adjust your count. Is this cheack neccessary?